Neos.Flow/Classes/Mvc/Routing/Router.php

```diff
@@ -80,17 +81,6 @@ class Router implements RouterInterface
      */
     protected $lastResolvedRoute;
 
-    /**
-     * Injects the (system) logger based on PSR-3.
-     *
-     * @param LoggerInterface $logger
-     * @return void
-     */
-    public function injectLogger(LoggerInterface $logger)
```

Comment:
Note: We have way too many tests depend on this method (and the other `injectLogger`). Would keeping the method interfere with the `@Flow\Inject` annotation?

Reply:
I was more questioning if I should maybe undo the removal (to help the tests). Otherwise need to replace all the `injectLogger(...)` calls there with `_set('logger', ...)` "magic"

Reply:
any thoughts on this @bwaidelich ?

Reply:
Okay, so I reverted this for now, then replaced the needless cases of `injectLogger()` in tests with `$this->inject()`. So this change is a little bit less breaking (if 3rd party depends on the `injectLogger()` method), yet we have the usage in core streamlined to where it's necessary.